apps/src/lib/kits/maker/redux.js

```diff
@@ -16,7 +16,7 @@ const MakerState = Immutable.Record({
 // Selectors
 function getRoot(state) {
   // Global knowledge eww
-  return state.maker;
+  return state.maker || new MakerState();
```

Comment:
This allows non-maker code to refer to Maker selectors safely, even in contexts where Maker isn't provided or Redux isn't set up (which is easy to do in storybook and is what was breaking the ApplabVisualizationColumn story).  The default MakerState reports maker disabled and everything disconnected.  All maker selectors go through this `getRoot()` helper so this makes them all work.